_includes/snippets/14/gcc_example.snip

```diff
@@ -1,73 +1,153 @@
-Let's take a closer look at the `gcc` module. GCC is an extremely widely used C/C++/Fortran
-compiler. Tons of software is dependent on the GCC version, and might not compile or run if the
-wrong version is loaded. In this case, there are two different versions: `gcc/4.8.5` and
-`gcc/5.4.0`. How do we load each copy and which copy is the default?
+To be more specific, you can specify the particular software you want.
+ e.g.
+ ```
+ {{ site.host_prompt }} module avail stata
+ ```
 
-In this case, `gcc/5.4.0` has a `(D)` next to it. This indicates that it is the default - if we type
-`module load gcc`, this is the copy that will be loaded.
+ {: .bash}
```

Comment:
This line should be on line 6, not 7, this applies to the code block above.

Reply:
See answer above about the bash and output anchors. I'm not sure about this.